config/config.go

```diff
@@ -19,15 +21,22 @@ func InitConfig() {
 	Host = os.Getenv("LN_SERVER_BASE_URL")
 	JwtKey = os.Getenv("LN_JWT_KEY")
 	RelayUrl = os.Getenv("RELAY_URL")
+	MemeUrl = os.Getenv("MEME_URL")
 	RelayAuthKey = os.Getenv("RELAY_AUTH_KEY")
+	RelayNodeKey = os.Getenv("RELAY_NODE_KEY")
```

Comment:
I'm not sure this Env var makes sense can we just use the `RELAY_AUTH_KEY` and `RELAY_URL` to hit a info endpoint on relay to get the pubkey, asking the server owner to hard code seems a bit easy to messup